src/librustc_passes/rvalue_promotion.rs

```diff
@@ -338,14 +338,14 @@ fn check_expr_kind<'a, 'tcx>(
                     if v.in_static {
                         for attr in &v.tcx.get_attrs(did)[..] {
                             if attr.check_name(sym::thread_local) {
-                                debug!("Reference to Static(id={:?}) is unpromotable \
-                                       due to a #[thread_local] attribute", did);
+                                debug!("Reference to `Static(id={:?})` is unpromotable \
```

Comment:
Maybe change all these and similar `debug!` expressions to begin with lower-case, as a matter of consistency/convention... not too important though. (And yes, this is pre-existing.)